pkgs/applications/virtualization/arion/arion.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs ? import ./. {}
+, arion-compose
+, arion-compose-eval-src
+}:
+
+let
+  inherit (pkgs) lib;
+  inherit (pkgs.haskell.lib) justStaticExecutables overrideCabal;
+
+  eval = args@{...}:
+    import
+      (arion-compose-eval-src + "/nix/eval-composition.nix")
+      ({ inherit pkgs; } // args);
+
+  build = args@{...}:
+    let composition = eval args;
+    in composition.config.out.dockerComposeYaml;
```

Comment:
Could you add comments on what these are supposed to be?  Also, an example of how they are intended to be used would be helpful.